components/zoho_crm/zoho_crm.app.mjs

```diff
@@ -35,13 +97,33 @@ module.exports = {
         headers,
       };
     },
+    _getAxiosParams(opts) {
```

Comment:
**-Minor-**
I'd suggest naming this method `_getRequestParams` or `_makeRequestConfig` so its intended usage can be understood by someone who doesn't know what axios is.